self-tests.yml

```diff
@@ -625,12 +596,12 @@ buildvariants:
   - name: race-detector
     display_name: Race Detector
     run_on:
-      - archlinux-new-small
-      - archlinux-new-large
+      - ubuntu1604-small
```

Comment:
Is there a reason that Evergreen would switch away from archlinux for the race detector? All the other repos use archlinux and if I remember correctly, the tests behave slightly differently on archlinux than on ubuntu.

If we're switching this, we have to switch all the other repos.

Reply:
That's interesting. How would it have benefited us to have the latest golang?
I'm game to update the tests on other repos. I'll open ticket(s), as Kim suggested.

Reply:
I think the idea was to have an early warning system about anything that might break in a newer version of Go. I don't remember that ever actually happening though.